2018 Robot/src/org/usfirst/frc/team4999/robot/OI.java

```diff
@@ -46,9 +45,8 @@
 	// until it is finished as determined by it's isFinished method.
 	// button.whenReleased(new ExampleCommand());
 	
-	Trigger liftPowerMonitor = new LiftPowerMonitor();
-	
+	Button moveXDistance = new JoystickButton(RobotMap.flightStick, 1);
 	public OI() {
-		liftPowerMonitor.whenActive(new KillLift());
+		moveXDistance.whenActive(new MoveDistance(5));
```

Comment:
Really not understanding what's going on here.  Please explain.

Reply:
It's the command-based robot's way of running commands when a button is pushed. [doc](https://wpilib.screenstepslive.com/s/currentCS/m/java/l/599739-running-commands-on-joystick-input)